core/cloudflow-operator/src/main/scala/cloudflow/operator/Name.scala

```diff
@@ -58,15 +64,27 @@ object Name {
   }
 
   /**
-   * Make a name compatible with DNS 1123 standard: like a full domain name.
+   * Makes a name compatible with DNS 1123 standard: like a full domain name, max 63 characters
    * Regex to follow: [a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*
-   * Limit the resulting name to 63 characters
+   * Limits the resulting name to 63 characters
+   * https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names
    */
-  private[operator] def makeDNS1123Compatible(name: String): String = {
+  private[operator] def makeDNS1123CompatibleLabelName(name: String): String = {
```

Comment:
This code is exclusively used in the tests now, can we remove this method and test the new `makeDNS1123CompatibleSubDomainName` instead?

Reply:
done.